packages/theming/src/utils/arrowStyles.ts

```diff
@@ -54,65 +36,77 @@ const animationStyles = (position: ArrowPosition, modifier: string) => {
     positionCss = css`
       top: ${placement};
-      right: ${position === 'top-right' && size};
-      left: ${position === 'top' ? '50%' : position === 'top-left' && size};
+      right: ${position === 'top-right' && _position};
+      left: ${position === 'top' ? '50%' : position === 'top-left' && _position};
       margin-left: ${position === 'top' && margin};
     `;
   } else if (position.startsWith('right')) {
-    propertyRadius = 'border-bottom-left-radius';
     clipPath = 'polygon(100% 0, 100% 100%, calc(100% - 1px) 100%, 0 1px, 0 0)';
+    borderCss = css`
+      border-bottom: none;
+      border-left: none;
+    `;
     positionCss = css`
-      top: ${position === 'right' ? '50%' : position === 'right-top' && size};
+      top: ${position === 'right' ? '50%' : position === 'right-top' && _position};
       right: ${placement};
-      bottom: ${position === 'right-bottom' && size};
+      bottom: ${position === 'right-bottom' && _position};
       margin-top: ${position === 'right' && margin};
     `;
   } else if (position.startsWith('bottom')) {
-    propertyRadius = 'border-top-left-radius';
     clipPath = 'polygon(100% 0, calc(100% - 1px) 0, 0 calc(100% - 1px), 0 100%, 100% 100%)';
+    borderCss = css`
+      border-top: none;
+      border-left: none;
+    `;
     positionCss = css`
-      right: ${position === 'bottom-right' && size};
+      right: ${position === 'bottom-right' && _position};
       bottom: ${placement};
-      left: ${position === 'bottom' ? '50%' : position === 'bottom-left' && size};
+      left: ${position === 'bottom' ? '50%' : position === 'bottom-left' && _position};
       margin-left: ${position === 'bottom' && margin};
     `;
   } else if (position.startsWith('left')) {
-    propertyRadius = 'border-top-right-radius';
     clipPath = 'polygon(0 100%, 100% 100%, 100% calc(100% - 1px), 1px 0, 0 0)';
+    borderCss = css`
+      border-top: none;
+      border-right: none;
+    `;
     positionCss = css`
-      top: ${position === 'left' ? '50%' : position === 'left-top' && size};
-      bottom: ${size};
+      top: ${position === 'left' ? '50%' : position === 'left-top' && _position};
+      bottom: ${_position};
       left: ${placement};
       margin-top: ${position === 'left' && margin};
     `;
   }
 
   /**
-   * 1. Round-off portion of the foreground square opposite the arrow tip
-   *    (improved layout for IE which doesn't support 'clip-path').
-   * 2. Clip portion of the foreground square opposite the arrow tip so that it
+   * 1. Clip portion of the foreground square opposite the arrow tip so that it
    *    doesn't interfere with container content.
+   * 2. Knock out border opposite the arrow tip.
    * 3. Arrow positioning on the base element.
    */
   return css`
     &::before {
-      ${propertyRadius!}: 100%; /* [1] */
-      clip-path: ${clipPath}; /* [2] */
+      clip-path: ${clipPath}; /* [1] */
+      ${borderCss}; /* [2] */
     }
 
     &::before,
     &::after {
```

Comment:
We might not even need an `::after` after all! 😉 🙌 

![Screenshot 2024-05-23 at 7 01 53 AM](https://github.com/zendeskgarden/react-components/assets/6879688/f96f8cc3-6642-4ab5-8e47-31bfacf3ec98)

Reply:
We need `::after` to retain the `box-shadow` inheritance. Otherwise, the arrow pokes out un-shadowed. Not terrible, but the original designs wanted the shadow to follow the arrow.

Reply:
Totally missed that but after some CSS tweaking I see it now. Thanks!

![Screenshot 2024-05-23 at 10 27 29 AM](https://github.com/zendeskgarden/react-components/assets/6879688/9275356f-ba79-4535-828c-88294e211384)